src/Workspaces/Core/Portable/FindSymbols/ReferenceLocation.cs

```diff
@@ -46,16 +48,22 @@ namespace Microsoft.CodeAnalysis.FindSymbols
         /// </summary>
         internal SymbolUsageInfo SymbolUsageInfo { get; }
 
+        /// <summary>
+        /// Additional properties for this reference
+        /// </summary>
+        internal ImmutableArray<FindUsageProperty> FindUsagesProperties { get; }
+
         public CandidateReason CandidateReason { get; }
 
-        internal ReferenceLocation(Document document, IAliasSymbol alias, Location location, bool isImplicit, SymbolUsageInfo symbolUsageInfo, CandidateReason candidateReason)
+        internal ReferenceLocation(Document document, IAliasSymbol alias, Location location, bool isImplicit, SymbolUsageInfo symbolUsageInfo, ImmutableArray<FindUsageProperty> additionalProperties, CandidateReason candidateReason)
```

Comment:
is this ever called directly by F#/TS?  if so, this is a binary break.

Reply:
I verified with TS and F# that they are not calling this.